test/test_argparse.py

```diff
@@ -12,7 +12,10 @@
 class TestParseItem(HPyTest):
     def make_parse_item(self, fmt, type, hpy_converter):
         mod = self.make_module("""
-            __attribute__((unused)) static inline
```

Comment:
maybe we should hide this difference behind a macro and put it in `hpy.h`. It looks like a generally useful feature.

Reply:
but hpy.h is not imported where the macro is needed

Reply:
yes, it's included automatically by `make_module`:
https://github.com/hpyproject/hpy/blob/d5c8442beae23c85d606f00e0f60ce816b1447dc/test/support.py#L56-L61